Gemfile

```diff
@@ -17,6 +17,8 @@ else
   gem 'journey'
 end
 
+gem 'i18n', '~> 0.6.11'
```

Comment:
Actually the problem is that the gem doesn't even install at those versions https://github.com/svenfuchs/i18n/blob/v0.7.0/i18n.gemspec#L21